src/components/BaseInput.vue

```diff
@@ -65,11 +68,15 @@ export default {
       border: 1px solid $cl-light-blue;
       background-color: $cl-blue;
     }
+
+    
   }
 }
 @media (min-width: 582px) {
   .base-input__input {
-    width: 472px;
+    width: 100%;
+    min-width: 472px;
```

Comment:
I don't know the context of this input width, but are you sure this input should be as wide as it wants to be? Some max-width could be useful, because we probably don't want to have input 1000px wide for example. Of course it might work in your layout, this is just a suggestion 😄

Reply:
I was trying to make some universal input, but I need to work on it...